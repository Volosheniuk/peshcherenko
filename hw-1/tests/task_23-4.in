w
-22
-11.5
-12.7
