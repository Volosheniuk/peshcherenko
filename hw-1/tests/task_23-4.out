w
-22
-11.500000
-12.700000
